### DISCUSSION TITLE
Media Control

### DISCUSSION THREAD

**Discussion by emrebilal** [ShankarBUS/ModernFlyouts#793] [General]
Media controls don't work this release I guess, but the progress bar that appears during music playback is slightly delayed. In Spotify, the bar moves, but there is no progress in the Groove Music app, the time is zero.

**Comment by prakharb5** [comment]
> Media controls don't work this release I guess

There are some reports of the media controls not working.

> the progress bar that appears during music playback is slightly delayed

What do you mean by the progress bar being delayed? Does it jump values at every 5 seconds?

> there is no progress in the Groove Music app, the time is zero

Groove Music does not support the timeline feature, hence it is stuck at 0.

**Reply by emrebilal** [reply]
Yes, it jump values at every 4-5 seconds, is this normal

**Reply by prakharb5** [reply]
Yes it is normal, since the apps provide the timeline update every 4-5 seconds. Even Microsoft advises to do this exact behaviour. No fix for it, sadly.
